.github/workflows/scheduled.yaml

```diff
@@ -99,19 +99,18 @@ jobs:
         id: setup
         run: |
           : Setup Environment
-          print '::group::Enable Xcode 15.2'
-          sudo xcode-select --switch /Applications/Xcode_15.2.app/Contents/Developer
+          print '::group::Enable Xcode 16.1'
+          sudo xcode-select --switch /Applications/Xcode_16.1.0.app/Contents/Developer
           print '::endgroup::'
 
-          local -a to_remove=()
+          local -a unwanted_formulas=(llvm gcc pkg-config@0.29.2)
 
-          for formula (llvm gcc postgresql openjdk sox libsndfile flac libvorbis opusfile \
-            libogg composer php gd freetype fontconfig webp libpng lame libtiff opus kotlin \
-            sbt libxft libxcb) {
-            if [[ -d /usr/local/opt/${formula} ]] to_remove+=(${formula})
+          local -a remove_formulas=()
+          for formula (${unwanted_formulas}) {
+            if [[ -d ${HOMEBREW_PREFIX}/Cellar/${formula} ]] remove_formulas+=(${formula})
```

Comment:
What's the reason for changing this list in this place but not the other?

Reply:
Copy+paste error I'd guess.